api/src/main/java/com/discordsrv/api/events/message/receive/game/AwardMessageReceiveEvent.java

```diff
@@ -96,4 +120,26 @@ public String toString() {
                 + "gameChannel=" + GameChannel.toString(gameChannel)
                 + '}';
     }
+
+    public enum AdvancementFrame {
+        TASK("task", new Color(0x55FF55)), // Green
+        GOAL("goal", new Color(0x55FF55)), // Green
+        CHALLENGE("challenge", new Color(0xAA00AA)); // Dark Purple
+
```

Comment:
Are these frames the only possible options, or is it possible to add new ones (via datapack or something else)?